DEV-17 Add course archiving (#1432)
CodeGra-de/CodeGra.de: docs/user/management.rst

Course Management
-------------------
On the first tab of the course management page, *General*, you can change the

> name of the course. The name should be at least one character. As the name of

Reviewer: ```suggestion
name of the course. The name should not be empty. As the name of
```

LTI courses is copied from the LMS, the name of LTI courses cannot be changed

> within CodeGrade. On this tab you can also archive a course. Archiving a course

Reviewer: ```suggestion
within CodeGrade.

On this tab you can also archive a course. Archiving a course
```
